Rlabkey/R/labkey.setCurlOptions.R

```diff
@@ -20,23 +20,23 @@ PACKAGE_ENV = new.env()
 labkey.setCurlOptions <- function(...)
 {
     # default curl options
-    options <- curlOptions(ssl.verifyhost=2, ssl.verifypeer=TRUE, followlocation=TRUE, sslversion=1L, useragent = "Rlabkey")
+    options <- config(ssl_verifyhost=2, ssl_verifypeer=TRUE, followlocation=TRUE, sslversion=1L, useragent = "Rlabkey")
 
     # check if a certificate bundle has been specified from the environment variable
     vars <- Sys.getenv("RLABKEY_CAINFO_FILE")
     if (nchar(vars[1]) > 0)
     {
-        options <- curlOptions(cainfo = vars[1], .opts=c(options))
+        options <- c(options, config(cainfo = vars[1]))
     }
 
     # merge in any overrides
-    options <- curlOptions(..., .opts=c(options))
+    options <- c(options, config(...))

```

Comment:
Karl, what did you decide for backwards compatibility here? Will this fail for any users that are currently using the dot notation for these params? Or should we explicitly check for that and warn/error?